setup/terraform/idp-cluster.tf

```diff
@@ -98,3 +98,61 @@ resource "humanitec_resource_definition_criteria" "cluster_local" {
     humanitec_resource_definition_criteria.agent
   ]
 }
+
+# Install and configure Humanitec Operator
+
+resource "kubernetes_namespace" "operator_namespace" {
+  metadata {
+    name = var.operator_ns
+  }
+}
+
+resource "tls_private_key" "operator" {
+  algorithm = "RSA"
+  rsa_bits  = 4096
+}
+
+resource "kubernetes_secret" "operator_private_key" {
+  depends_on = [kubernetes_namespace.operator_namespace]
+  metadata {
+    name      = "humanitec-operator-private-key"
+    namespace = var.operator_ns
+  }
+  data = {
+    privateKey              = tls_private_key.operator.private_key_pem
+    humanitecOrganisationID = var.humanitec_org
+  }
+}
+
+resource "humanitec_key" "operator_public_key" {
+  depends_on = [kubernetes_secret.operator_private_key]
+  key = tls_private_key.operator.public_key_pem
+}
+
+resource "kubernetes_namespace" "secret_store_namespace" {
+  metadata {
+    name = "5min-idp-secrets"
+  }
+}
+
+resource "helm_release" "operator" {
+  depends_on = [
+    kubernetes_secret.operator_private_key,
+    kubernetes_namespace.secret_store_namespace,
+    humanitec_key.operator_public_key
+  ]
+  name       = "humanitec-operator"
+  namespace  = var.operator_ns
+  repository = "oci://ghcr.io/humanitec/charts"
+  chart      = "humanitec-operator"
+}
+
+// Hack: registering fake "vault" primary store just to enable Operator Mode ("kubernetes" type is not supported in Orchestrator).
+// Needs to be removed, when Operator Mode becomes the default one.
+resource "humanitec_secretstore" "kubernetes_secret_store" {
+  id      = "5min-idp-secrets"
+  primary = true
+  vault = {
+    url = "https://example.com"
+  }
+}
```

Comment:
This app doesn't have secrets, so we need to "force" Operator Mode with a primary store.

Reply:
Not yet, we wait for exactly this change to introduce the FF flag, and then we can remove it this part. Step-by-step not to break anything in the middle.